pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -110,7 +110,7 @@ rustPlatform.buildRustPackage (finalAttrs: {
     owner = "zed-industries";
     repo = "zed";
     tag = "v${finalAttrs.version}";
-    hash = "sha256-5bn70R5phHZmAf/71IK8o2laHcCo92cCiaKQ7xo0Uag=";
+    hash = "sha256-STMDbUMsQAPJCDJMFFpb18fBtzOjxvhvsqx94MpCTac=";
```

Comment:
you should also update `cargoHash = "sha256-NbjoaHZpqjWz6AnuE9EXAMuyeXldlm4QB765fp6QVlI=";` :see_no_evil:

Reply:
Thanks!